src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
Great job @elafildecolor 🤝

- [ ] As stated in the project requirements, kindly make sure the `App.js` component is class-based.